plenoptic/tools/conv.py

```diff
@@ -7,87 +7,129 @@
                                         dilation=1, groups=n_channels)
 
         elif edges == 'reflect1':
             pad = nn.ReflectionPad1d(filt.shape[-1]//2)
-            return nn.functional.conv1d(pad(image), filt, bias=None, stride=step,
-                                        padding=0, dilation=1, groups=n_channels)
+            return nn.functional.conv1d(pad(signal), filt, bias=None,
+                                        stride=step, padding=0, dilation=1,
+                                        groups=n_channels)
 
-    if len(image.shape) == 4:
+    if len(signal.shape) == 4:
 
         if isinstance(filt, np.ndarray) or filt.shape[0] != n_channels:
-            filt = torch.tensor(filt, dtype=torch.float32).repeat(n_channels,  1, 1, 1).to(image.device)
+            filt = torch.tensor(filt, dtype=torch.float32)
+            filt = filt.repeat(n_channels,  1, 1, 1).to(signal.device)
 
         if edges == 'zero':
-            return nn.functional.conv2d(image, filt, bias=None, stride=step,
-                                        padding=(filt.shape[-2] // 2, filt.shape[-1] // 2),
+            return nn.functional.conv2d(signal, filt, bias=None, stride=step,
+                            padding=(filt.shape[-2] // 2, filt.shape[-1] // 2),
                                         dilation=1, groups=n_channels)
 
         elif edges == 'reflect1':
             pad = nn.ReflectionPad2d(filt.shape[-1]//2)
-            return nn.functional.conv2d(pad(image), filt, bias=None, stride=step,
-                                        padding=0, dilation=1, groups=n_channels)
+            return nn.functional.conv2d(pad(signal), filt, bias=None,
+                                        stride=step, padding=0, dilation=1,
+                                        groups=n_channels)
 
-    if len(image.shape) == 5:
+    if len(signal.shape) == 5:
 
         edges = 'zero'
         step = (2, 2, 2)
 
-
         if isinstance(filt, np.ndarray) or filt.shape[0] != n_channels:
-            filt = torch.tensor(filt, dtype=torch.float32).repeat(n_channels, 1, 1, 1, 1)
+            filt = torch.tensor(filt, dtype=torch.float32)
+            filt = filt.repeat(n_channels, 1, 1, 1, 1)
 
         if edges == 'zero':
-            return nn.functional.conv3d(image, filt, bias=None, stride=step,
-                                        padding=(filt.shape[-3] // 2, filt.shape[-2] // 2, filt.shape[-1] // 2),
+            return nn.functional.conv3d(signal, filt, bias=None, stride=step,
+            padding=(filt.shape[-3] // 2, filt.shape[-2] // 2, filt.shape[-1] // 2),
                                         dilation=1, groups=n_channels)
 
         # elif edges == 'reflect1':
         #     pad = nn.ReflectionPad3d(filt.shape[-1] // 2)
-        #     return nn.functional.conv3d(pad(image), filt, bias=None, stride=step,
+        #     return nn.functional.conv3d(pad(signal), filt, bias=None, stride=step,
```

Comment:
if we're not using this, it should be removed

Reply:
agreed, see #126